Sources/FoundationEssentials/WinSDK+Extensions.swift

```diff
@@ -288,4 +288,29 @@ internal func WIN32_FROM_HRESULT(_ hr: HRESULT) -> DWORD {
     return DWORD(hr)
 }
 
+internal func Win32FillNullTerminatedStringBuffer(initialSize: DWORD, maxSize: DWORD, _ attempt: (UnsafeMutableBufferPointer<WCHAR>) throws -> DWORD) throws -> String {
```

Comment:
I think that we should include `Wide` in the name. I would also be okay with a `W` suffix. I'm not sure that the Win32 prefix matches any of the other helpers that we have though.

I think that `body` is better than `attempt` as a parameter name.

Reply:
Done